xcode/Subconscious/Shared/Components/Common/SubtextTextViewRepresentable.swift

```diff
@@ -164,38 +164,55 @@ struct SubtextTextViewRepresentable: UIViewRepresentable {
         }
     }
 
-    class SubtextTextParagraph: NSTextParagraph {
-        private(set) var subtext: Subtext
-
-        init(attributedString: NSAttributedString) {
-            SubtextTextViewRepresentable.logger.debug(
-                "SubtextTextParagraph: parse and render markup"
-            )
-            let subtext = Subtext(markup: attributedString.string)
-            let renderedAttributedString = subtext.renderAttributedString(
-                url: SubtextTextViewRepresentable.toSubURL
-            )
-            self.subtext = subtext
-            super.init(attributedString: renderedAttributedString)
-        }
-    }
-
     //  MARK: Coordinator
-    class Coordinator: NSObject, UITextViewDelegate, NSTextContentStorageDelegate, NSTextContentManagerDelegate, NSTextLayoutManagerDelegate {
+    class Coordinator: NSObject, UITextViewDelegate, NSTextContentStorageDelegate, NSTextStorageDelegate, NSTextContentManagerDelegate, NSTextLayoutManagerDelegate {
```

Comment:
Nit: let's wrap this long line. https://google.github.io/swift/#type-and-extension-declarations